packages/asc-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -38,18 +42,37 @@ function formatPanelId(id: string) {
   return `panel-${id}`
 }
 
-export const Tabs: React.FC<TabsProps> = ({ label, children }) => {
+export const Tabs: React.FC<TabsProps> = ({
+  label,
+  children,
+  initialTab,
+  ...otherProps
+}) => {
   const allTabs = children.map(({ props }) => props.id)
-  const firstTab = allTabs[0] ?? ''
-  const [selectedTab, setSelectedTab] = useState(firstTab)
+  const foundInitialTab = allTabs.find((id) => id === initialTab)
+  const activeTab = foundInitialTab ?? allTabs[0] ?? ''
+
+  if (initialTab && !foundInitialTab) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      `Warning: you passed a wrong initialTab value to Tabs component. Given initialTab ID: ${initialTab}, length of tabs: ${allTabs.length} `,
+    )
+  }
+
+  const [selectedTab, setSelectedTab] = useState(activeTab)
   const tabListRef = useRef<HTMLDivElement>(null)
   const { keyDown } = useFocusWithArrows(tabListRef, true, true, true)
 
   return (
     <>
-      <TabList aria-label={label} onKeyDown={keyDown} ref={tabListRef}>
+      <TabList
+        aria-label={label}
+        onKeyDown={keyDown}
+        ref={tabListRef}
+        {...otherProps}
```

Comment:
Careful with these props, as now the `onKeyDown` can be overwritten by the user. If we allow users to pass these kind of props we need to be able to proxy these events, values and references as well.